cbw_api_toolbox/cbw_api.py

```diff
@@ -505,3 +507,77 @@ def upload_importer_results(self, content):
+        response = self._get_pages("GET", [ROUTE_COMPLIANCE_ASSETS], params)
+        return response
+
+    def compliance_server(self, server_id):
+        """GET request to /api/v3/compliance/servers/{server_id} to get all informations
+        about a specific compliance asset"""
+        response = self._request("GET", [ROUTE_COMPLIANCE_ASSETS, server_id])
+        if response.status_code != 200:
+            logging.error("Error server id::{}".format(response.text))
+            return None
+        return self._cbw_parser(response)
+
+    def recheck_rules(self, server_id):
+        """PUT request to /api/v3/compliance/servers/{server_id}/recheck_rules to recheck
+        the rules of a specific compliance asset"""
+        response = self._request("PUT", [ROUTE_COMPLIANCE_ASSETS, server_id, 'recheck_rules'])
+        if response.status_code != 200:
+            logging.error("Error server id::{}".format(response.text))
+            return None
+        return self._cbw_parser(response)
+
+    def compliance_rules(self, params=None):
+        """GET request to /api/v3/compliance/rules to get all rules from compliance"""
+        response = self._get_pages("GET", [ROUTE_COMPLIANCE_RULES], params)
+        return response
+
+    def compliance_rule(self, server_id):
+        """GET request to /api/v3/compliance/rules/{server_id} to get all rules
+        tested on a specific asset"""
+        response = self._request("GET", [ROUTE_COMPLIANCE_RULES, server_id])
+        if response.status_code != 200:
+            logging.error("Error server id::{}".format(response.text))
+            return None
+        return self._cbw_parser(response)
+
+    def create_compliance_rule(self, params):
+        """POST request to /api/v3/compliance/rules to create a compliance rule"""
+        response = self._request("POST", [ROUTE_COMPLIANCE_RULES], params)
+        if response.status_code != 201:
+            logging.error("Error::{}".format(response.text))
+            return None
+
+        return self._cbw_parser(response)
+
+    def update_compliance_rule(self, rule_id, info):
+        """PUT request to /api/v3/compliance/rules/{rule_id} to update the groups of a server"""
+        if rule_id:
+            response = self._request("PUT", [ROUTE_COMPLIANCE_RULES, rule_id], info)
+            logging.info("Update compliance rule with: {}".format(info))
+            return self.verif_response(response)
+
+        logging.error("No compliance rule id for update")
+        return False
+
+    def delete_compliance_rule(self, rule_id):
+        """DELETE request to /api/v3/compliance/rules/{rule_id} to delete a compliance rule"""
+        if rule_id:
+            logging.debug("Deleting {}".format(rule_id))
+            response = self._request("DELETE", [ROUTE_COMPLIANCE_RULES, rule_id])
+            return self.verif_response(response)
+
+        logging.error("No rule id specific for delete")
```

Comment:
This is not english. 
create an issue so that we fix it everywhere, and fix it from this mr please.

Reply:
https://github.com/Cyberwatch/cyberwatch_api_toolbox/issues/199